app/helpers/imports_helper.rb

```diff
@@ -0,0 +1,9 @@
+module ImportsHelper
+  def progress(podcast_import)
```

Comment:
progress bar logic. took a long time to figure out that i needed to convert to floats to get accurate percentages.